### PR TITLE
Add a new storage procedure to migration guide.

### PR DESCRIPTION
**Pull request by jpmedley**
Added this bit of conversion wisdom to the migration guide so that we can get migration material out of [storage documentation](https://developer.chrome.com/docs/extensions/reference/storage/#overview).

No Bug

### CODE REVIEW

**Comment by jpmedley** [comment]
Pinging on this. This would let you remove some duplication in your privacy sandbox stuff.

File: site/en/docs/extensions/migrating/to-service-workers/index.md [hunk 1]
```diff
@@ -109,10 +109,17 @@ Communicate between offscreen documents and extension service workers using [mes
...
+The web platform's [`Storage`](https://developer.mozilla.org/docs/Web/API/Storage) interface (accessible from `window.localStorage`) cannot be used in a service worker. To address this do one of two things. First, you can replace it with calls to another storage mechanism. The [`chrome.storage.local`](/docs/extensions/reference/storage/#property-local) namespace will serve most use cases, but [other options](/docs/extensions/mv3/service_workers/service-worker-lifecycle/#persist-data) are available.
 
-* Move its calls to an [offscreen document](/docs/extensions/reference/offscreen/).
-* Replace it with calls to another storage mechanism. The [`chrome.storage.local`](/docs/extensions/reference/storage/#property-local) namespace will serve most use cases. [Other options](/docs/extensions/mv3/service_workers/service-worker-lifecycle/#persist-data) are available.
+You can also move its calls to an [offscreen document](/docs/extensions/reference/offscreen/).
+
+1. Create an offscreen document with a conversion routine and a [`runtime.onMessage`](/docs/extensions/reference/runtime/#event-onMessage) handler.
```
Comment (oliverdunk) [inline site/en/docs/extensions/migrating/to-service-workers/index.md:116]: I think the lead-in sentence here makes it sound like we're going to give general advice, but then the steps are very much a migration routine. Could we either:

- Add a sentence before along the lines of "For example, to migrate data previously stored in localStorage to another mechanism:"
- Replace this with an actual long term way of doing storage, e.g starting an offscreen document, sending some data and then storing it. And then similar to read data.

Hope that makes sense :)

Comment (oliverdunk) [reply site/en/docs/extensions/migrating/to-service-workers/index.md:116]: Given that this is a migration page I think the former would be more useful, but I'm ok with either.

File: site/en/docs/extensions/migrating/to-service-workers/index.md [hunk 2]
```diff
@@ -109,10 +109,17 @@ Communicate between offscreen documents and extension service workers using [mes
...
 
-The web platform's [`Storage`](https://developer.mozilla.org/docs/Web/API/Storage) interface (accessible from `window.localStorage`) cannot be used in a service worker. To address this do one of the following:
+The web platform's [`Storage`](https://developer.mozilla.org/docs/Web/API/Storage) interface (accessible from `window.localStorage`) cannot be used in a service worker. To address this do one of two things. First, you can replace it with calls to another storage mechanism. The [`chrome.storage.local`](/docs/extensions/reference/storage/#property-local) namespace will serve most use cases, but [other options](/docs/extensions/mv3/service_workers/service-worker-lifecycle/#persist-data) are available.
 
-* Move its calls to an [offscreen document](/docs/extensions/reference/offscreen/).
-* Replace it with calls to another storage mechanism. The [`chrome.storage.local`](/docs/extensions/reference/storage/#property-local) namespace will serve most use cases. [Other options](/docs/extensions/mv3/service_workers/service-worker-lifecycle/#persist-data) are available.
+You can also move its calls to an [offscreen document](/docs/extensions/reference/offscreen/).
```
Comment (jpmedley) [inline site/en/docs/extensions/migrating/to-service-workers/index.md:114]:
```suggestion
You can also move its calls to an [offscreen document](/docs/extensions/reference/offscreen/). For example, to migrate data previously stored in `localStorage` to another mechanism:
```

File: site/en/docs/extensions/migrating/to-service-workers/index.md (hunk 1 above)
Comment (jpmedley) [reply site/en/docs/extensions/migrating/to-service-workers/index.md:116]: Done.

**Comment by jpmedley** [comment]
PTAL

**Review by oliverdunk — APPROVED** [review]
LGTM.

**Review by AmySteam — APPROVED** [review]

**Comment by sebastianbenz** [comment]
@jpmedley can we merge this?

**Comment by jpmedley** [comment]
The build still hasn't finished. That must be why I forgot it. Let me try resubmitting.

**Pull request closed without merging**